steamcmd_servers/quake_live/egg-quake-live.json

```diff
@@ -0,0 +1,82 @@
+    "description": "Quake Live is a first-person shooter video game by id Software. It is an updated version of Quake III Arena that was originally designed as a free-to-play game launched via a web browser plug-in. On September 17, 2014, the game was re-launched as a standalone title on Steam.",
+    "features": null,
+    "image": "quay.io\/parkervcp\/pterodactyl-images:ubuntu_source",
+    "startup": ".\/qzeroded.x64  +set net_port \"{{SERVER_PORT}}\" +set sv_hostname \"{{SERVER_NAME}}\" +set sv_serverType \"{{SERVER_TYPE}}\" +sv_maxClients \"{{MAX_CLIENTS}}\" +set g_password \"{{PASSWORD}}\" +sv_fps \"{{FPS}}\"",
+    "config": {
+        "files": "{}",
+        "startup": "{\r\n    \"done\": \"zmq stats and rcon passwords updated\",\r\n    \"userInteraction\": []\r\n}",
+        "logs": "{\r\n    \"custom\": false,\r\n    \"location\": \"logs\/latest.log\"\r\n}",
+        "stop": "quit"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\r\n# Quake Live Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\napt -y update\r\napt -y --no-install-recommends install curl lib32gcc1 ca-certificates libstdc++6 libz1\r\n\r\ncd \/tmp\r\ncurl -sSL -o steamcmd.tar.gz http:\/\/media.steampowered.com\/installer\/steamcmd_linux.tar.gz\r\n\r\nmkdir -p \/mnt\/server\/steamcmd \/mnt\/server\/logs\r\ntar -xzvf steamcmd.tar.gz -C \/mnt\/server\/steamcmd\r\ncd \/mnt\/server\/steamcmd\r\n\r\n# SteamCMD fails otherwise for some reason, even running as root.\r\n# This is changed at the end of the install process anyways.\r\nchown -R root:root \/mnt\r\n\r\nexport HOME=\/mnt\/server\r\n.\/steamcmd.sh +login anonymous +force_install_dir \/mnt\/server +app_update 349090 +quit\r\n\r\nmkdir -p \/mnt\/server\/.steam\/sdk32\r\ncp -v linux32\/steamclient.so ..\/.steam\/sdk32\/steamclient.so",
+            "container": "ubuntu:18.04",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "ld lib path",
+            "description": "",
+            "env_variable": "LD_LIBRARY_PATH",
+            "default_value": ".\/linux64",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|max:20"
+        },
+        {
+            "name": "Server Name",
+            "description": "The Name that will show up in the Serverbrowser",
+            "env_variable": "SERVER_NAME",
+            "default_value": "A Quake Live Server",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:70"
+        },
+        {
+            "name": "Server Type",
+            "description": "0 = Offline, 1 = LAN, 2 = Internet",
+            "env_variable": "SERVER_TYPE",
+            "default_value": "2",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|numeric|max:2"
+        },
+        {
+            "name": "Max Clients",
```

Comment:
As I have called out in many other submitted games I don't like limiting this by default and should be user editable at least if it's required.

Most people use pterodactyl to self-host servers. Making them go to the admin pages is limiting and annoying.

Reply:
Beyond the max players I think the rest looks fine.

Reply:
done